docs/learn/ticks-and-update-rates.md

```diff
@@ -1,58 +1,39 @@
+For example, if an in-game gun is capable of shooting more times per second than the client can update, then at some point the impact of more than one bullet will be communicated in the same update. For players receiving this update, it will seem as if a single bullet is doing more damage than it should, because they're actually receiving updates from multiple bullets at once.
 
-RPM vs Update Rates: "Super Bullets" damage of multiple hits sent in one update:
+The diagram below illustrates the difference between a 10 Hz update rate and a 60 Hz update rate. With a 10 Hz update rate, the game sends updates to the server ten times a second, so a gun that can fire 600 rounds per minute (RPM), or ten times a second, will only ever send a single bullet per update. A gun that fires at 750 RPM, more than ten times a second, will end up running into the super bullet problem. A higher update rate of 60 Hz resolves this issue, ensuring that there are enough updates for each bullet to arrive separately in its own update.
 
-<ImageSwitcher 
+<ImageSwitcher
 lightImageSrc="/rpm_update_rates-light.png?text=LightMode"
 darkImageSrc="/rpm_update_rates-dark.png?text=DarkMode"/>
 
-## Tick or Simulation Rate
-
-The tick rate, or simulation rate, tells us how many times per second the game produces and processes data.
-
-At the beginning of a tick, the server starts to process the data it received and runs its simulations. Then it sends the result to the clients and sleeps until the next tick happens. The faster the server finishes a tick the earlier the clients will receive new data from the server, which reduces the delay between player and server. That leads to a more responsive hit registration. 
-
-Tick rate: Simulation, tick processing:
-
-![Tick rate](/img/tick_rate.png)
-
-A tick or simulation rate of 60Hz will cause less delay than a tick rate of 30Hz, as it decreases the time between the simulation steps. A tick rate of 60Hz will also allow the server to send 60 updates per second, which compared to 30Hz reduces the round trip delay between the client and the server by about 33ms (-16ms from the client to the server, and another -16ms from the server to the client).
+### Discrepancy between tick rate and update rate
 
-When a server gets close to the limit, or even fails to process a tick inside that timeframe, then you will instantly notice the results: all sorts of strange gameplay issues like rubber banding, players teleporting, hits getting rejected, and physics failing.
+If the update rate of a client is lower than the tick rate of the server, then the client won't see the benefit of the high tick rate, because it will only receive updates at its update rate, even if multiple ticks have been processed in the interim.
```

Comment:
```suggestion
If the update rate of a client is lower than the tick rate of the server, then the client won't see the benefit of the high tick rate, because it will only receive updates at the client update rate, even if multiple ticks have been processed in the interim.
```

Reply:
just to make it 100 percent clear.